### ISSUE TITLE
Implement movement system

### ISSUE THREAD

**Issue by altered-beast** [CHATALOT1/team-pastel-sky-piratejam#7]
the player should be able to move the character with the WASD keys. ideally the player should be able to change the exact keys for the sake of accessibility.
this should also take into account collisions with walls and moving between the two floors of the level

**Comment by CHATALOT1** [comment]
Considering the time constraints and the amount of work required to create a settings UI, do we need to make the keys rebindable?

**Comment by altered-beast** [comment]
we dont need to no, but if we have the time spare at any point it would be nice to
